aion/web/elm/src/Update.elm

```diff
@@ -468,10 +471,14 @@ update msg model =
                 )
 
         ReceiveDisplayQuestion _ ->
-            { model | roomState = QuestionDisplayed } ! []
+            { model
+                | roomState = QuestionDisplayed
+                , progressBar = model.progressBar |> withProgress 0 |> withRunning Uninitialized |> withStart 0
+            }
+                ! [ Delay.after progressBarDelay millisecond Tick ]
 
         ReceiveQuestionBreak raw ->
-            { model | roomState = QuestionBreak } ! []
+            { model | roomState = QuestionBreak, progressBar = model.progressBar |> withRunning Stopped |> withProgress 0 } ! []
```

Comment:
Kind of inconsistent formatting, above you have `| roomState = ...` and `progressBar = ...` in separate lines.

Reply:
True, I would switch to one model-entry per line, like above. Does that sound ok?